server/datastore/mysql/migrations/tables/20240813103800_SupportMultipleABMTokens.go

```diff
@@ -0,0 +1,192 @@
+package tables
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
+	"github.com/google/uuid"
+	"github.com/jmoiron/sqlx"
+	"github.com/jmoiron/sqlx/reflectx"
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240813103800, Down_20240813103800)
+}
+
+func Up_20240813103800(tx *sql.Tx) error {
+	_, err := tx.Exec(`
+CREATE TABLE abm_tokens (
+	id                     int(10) UNSIGNED NOT NULL AUTO_INCREMENT,
+	organization_name      varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
+	apple_id               varchar(255) COLLATE utf8mb4_unicode_ci NOT NULL,
+	terms_expired          tinyint(1) NOT NULL DEFAULT '0',
+	renew_at               timestamp NOT NULL,
+	token                  blob NOT NULL,
```

Comment:
`blob` allows 64KB which I believe is more than enough for ABM tokens - small JSON objects with this payload: https://github.com/fleetdm/fleet/blob/f78761dbf6480de3a89c1765d027ee3770e4fb78/server/mdm/nanodep/client/auth.go#L50-L56

Reply:
sanity check: okay to store this unencrypted?

Reply:
No, I don't think we want that! Good think to double-check, it gets copied over from `mdm_config_assets`, and I believe it is encrypted in that table, right?

Reply:
(the struct I pasted above was just to give an idea of the size of the token, but it would be that JSONified object, encrypted, as it is today in `mdm_config_assets`).

Reply:
it is, thanks, that makes sense!